### DISCUSSION TITLE
Change locale on run time

### DISCUSSION THREAD

**Discussion by mdellanave** [apexcharts/apexcharts.js#2932] [Q&A]
Hi, is it possible to change the locale on run time?

I tried using the setLocale method but my area chart datetime axis doen't refresh with new labels

Am i missing something?

https://jsfiddle.net/1Lmwt6us/10/

**Comment by unzsnu** [comment]
Hi there, maybe it's a bit late :)  
You have to call update, after setting the new locale. 

`
chart.setLocale('it');
chart.update();
`
